apps/site-app/src/containers/ContactUs/index.tsx

```diff
@@ -68,12 +67,12 @@ export const ContactUs = () => {
               />
               </div>
             </div>
-            <UserMessage 
+            {/*<UserMessage 
```

Comment:
Commented that message to avoid giving the impression that a message was already sent with the user account